src/model_repository_manager.cc

```diff
@@ -109,12 +110,26 @@ class LocalizeRepoAgent : public TritonRepoAgent {
                      "' must have bytes type for its value")
                         .c_str());
               }
-
-              // Save model file to the instructed directory
-              // mkdir
+              // Save model override file to the instructed directory using the
+              // temporary model directory as the basepath.
               const std::string file_path =
                   JoinPath({temp_dir, file->Name().substr(file_prefix.size())});
               const std::string dir = DirName(file_path);
```

Comment:
small nit:

instead of creating new variables, could we update the existing?
```
const std::string & dir = boost::filestystem::weakly_canonical(DirName(file_path)).string();

const std::string &temp_dir = boost::filessystem::weakly_canonical(temp_dir_cstr).string();

```